weave

```diff
@@ -1717,13 +1773,26 @@ launch_router() {
             --no-restart)
                 RESTART_POLICY=
                 ;;
+            --awsvpc)
+                WEAVE_NO_FASTDP=1
+                AWSVPC_ARGS="--awsvpc"
+                AWSVPC=1
+                ;;
             *)
                 ARGS="$ARGS '$(echo "$1" | sed "s|'|'\"'\"'|g")'"
                 ;;
         esac
         shift
     done
     eval "set -- $ARGS"
+
+    create_bridge
+    docker_bridge_ip
```

Comment:
Do you mean that it breaks this line https://github.com/weaveworks/weave/pull/2091/files#diff-ed6579eaedd0e67b3ed51c7ee7071044R1735 ?

If yes, I don't see how it breaks. `CONTAINER_PORT` is used only when starting the router.